docker-compose.yml

```diff
@@ -22,6 +24,45 @@ services:
       INFLUXDB_DB: openwisp2
       INFLUXDB_USER: openwisp
       INFLUXDB_USER_PASSWORD: openwisp
+  # clustered version of elasticsearch is used as that might be used in production
+  es01:
```

Comment:
We don't need to run Elasticsearch in a High Available environment. Testing HA is their job :smiley: . We can simply make sure that setting up a multi-nodes cluster works but IMHO it is enough for us to run tests in only one instance.

WDYT?

Reply:
I agree though there are some problems that I was facing with elasticsearch docker due to which too I am using two nodes :sweat_smile:. Can you please check out if it's possible to run elasticsearch on a single port (I am not sure about this as I could not :/ ) and then I can adapt. Thanks!